frameworks/cassandra/docs/release-notes.md

```diff
@@ -7,6 +7,19 @@ menuWeight: 120
 
 ---
 
+## Version 2.1.2-3.0.15-beta
+
+## NOTICE
+
+This is a beta release of the DC/OS Apache Cassandra framework. It contains multiple improvements as well as new features that are to be considered of beta quality. Do _not_ operate this version in production.
+
+### Improvements
```

Comment:
In the other docs we use `Updates`?

Also, was there a Health-check change?

Reply:
I think we haven't been very consistent...

```
./frameworks/elastic/docs/release-notes.md:49:### Improvements
./frameworks/elastic/docs/release-notes.md:59:### Improvements
./frameworks/elastic/docs/release-notes.md:69:### Improvements
./frameworks/elastic/docs/release-notes.md:94:### Improvements/Features
./frameworks/template/docs/release-notes.md:11:### Improvements
./frameworks/kafka/docs/release-notes.md:42:### Improvements
./frameworks/kafka/docs/release-notes.md:56:### Improvements
./frameworks/cassandra/docs/release-notes.md:16:### Improvements
./frameworks/cassandra/docs/release-notes.md:54:### Improvements
./frameworks/cassandra/docs/release-notes.md:68:### Improvements
./frameworks/hdfs/docs/release-notes.md:51:### Improvements
./frameworks/hdfs/docs/release-notes.md:64:### Improvements
```

```
./frameworks/elastic/docs/release-notes.md:16:### Updates
./frameworks/elastic/docs/release-notes.md:35:### Updates
./frameworks/kafka/docs/release-notes.md:37:### Updates
./frameworks/cassandra/docs/release-notes.md:29:### Updates
./frameworks/cassandra/docs/release-notes.md:48:### Updates
./frameworks/hdfs/docs/release-notes.md:39:### Updates
```

> Also, was there a Health-check change?

Yes! I'll add it, thanks.

Reply:
1c128bf0bc6b7f3d7940ee4c654ac256b83644f0

Reply:
IIRC...

Bug Fixes
Improvements
Breaking Changes